internal/pkg/agent/application/info/state.go

```diff
@@ -28,11 +31,33 @@ func RunningInstalled() bool {
 	return true
 }
 
-func CreateInstallMarker(topPath string) error {
+func CreateInstallMarker(topPath string, uidStr string, gidStr string) error {
 	markerFilePath := filepath.Join(topPath, MarkerFileName)
 	if _, err := os.Create(markerFilePath); err != nil {
 		return err
 	}
 
+	var err error
+	if runtime.GOOS != "windows" {
```

Comment:
Since `os.Getuid()` and `os.Getgid()` do actually work on Windows, it's not immediately obvious (without the context of this PR) why we need this `if` condition here.  Consider adding an explanatory comment.